packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/container.test.tsx

```diff
@@ -105,6 +106,10 @@ function resetMockSensitiveDataClient() {
   (useSensitiveDataClient as jest.Mock).mockImplementation(
     () => mockSensitiveDataClient
   );
+  mockSensitiveDataClient.getDataType = jest.fn().mockReturnValue({
+    keyFetchToken: MOCK_KEY_FETCH_TOKEN,
+    unwrapBKey: MOCK_UNWRAP_BKEY,
+  });
```

Comment:
It's rather unfortunate that we have to do this here since we create a real SensitiveDataClient, but then we mock out the set/get so we can assert the function call? Wouldn't it be better to assert the state change instead (i.e. check if the Auth data was added into the real client)?